cogs5e/models/automation.py

```diff
@@ -947,7 +954,7 @@ def run(self, autoctx):
                 autoctx.queue(f"**Concentration**: dropped {', '.join([e.name for e in conc_conflict])}")
         else:
             effect = init.Effect.new(None, None, self.name, duration, autoctx.parse_annostr(self.effects),
-                                     tick_on_end=self.tick_on_end, concentration=self.concentration)
+                                     tick_on_end=self.tick_on_end, concentration=self.concentration, desc=self.desc)
```

Comment:
should this be `desc` rather than `self.desc` to get the rendered description?

Reply:
Yes, yes it should be